Modules/_struct.c

```diff
@@ -497,9 +497,8 @@ nu_ulonglong(_structmodulestate *state, const char *p, const formatdef *f)
 static PyObject *
 nu_bool(_structmodulestate *state, const char *p, const formatdef *f)
 {
-    _Bool x;
-    memcpy(&x, p, sizeof x);
-    return PyBool_FromLong(x != 0);
+    const _Bool bool_false = false;
```

Comment:
```suggestion
    const _Bool bool_false = 0;
```

Or use and `false` and `bool` (not `_Bool`) , but then include `<stdbool.h>` directly, don't rely on something else pulling it in. These will become C keywords in C23 but we aren't there yet.

Reply:
Ah, I thought it's included in Python.h like stdint.h.